callisto/delivery/api.py

```diff
@@ -20,33 +43,25 @@ class DeliveryApi(object):
         DeliveryApi maps to
     '''
 
-    def __init__(self):
-        self.set_api_class()
+    api_env_variable = 'CALLISTO_NOTIFICATION_API'
+    default_classpath = 'callisto.delivery.api.AbstractNotification'
```

Comment:
should this default to the implementation in notifications so it works without a settings var?